plugins/xml-element-extractor/scripts/extract_xml_element.py

```
@@ -0,0 +1,248 @@
+        return f'<{tag_name}>'
+
+
+def extract_xml_element(source_file, dest_file, element_tag):
+    """Extract XML element from source file to dest file."""
+    try:
+        print(f"Extracting element: {element_tag}")
+        print(f"From: {source_file}")
+        print(f"To: {dest_file}")
+
+        # Parse the XML file
+        tree = ET.parse(source_file)
```

Ruff (0.14.3)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)